pandas/_libs/groupby_helper.pxi.in

```diff
@@ -571,10 +572,16 @@ def group_rank_{{name}}(ndarray[float64_t, ndim=2] out,
             # (used by pct calculations later). also be sure to reset any of
             # the items helping to calculate dups
             if i == N - 1 or labels[_as[i]] != labels[_as[i+1]]:
-                for j in range(grp_start, i + 1):
-                    grp_sizes[_as[j], 0] = i - grp_start + 1 - grp_na_count
+                if tiebreak != TIEBREAK_DENSE:
+                    for j in range(grp_start, i + 1):
+                        grp_sizes[_as[j], 0] = i - grp_start + 1 - grp_na_count
+                else:
+                    for j in range(grp_start, i + 1):
+                        grp_sizes[_as[j], 0] = (grp_tie_count
```

Comment:
minor, but can you put the ``-`` on the previous line

Reply:
break line after the operator ?
```
                    for j in range(grp_start, i + 1):
                        grp_sizes[_as[j], 0] = (grp_tie_count -
                                                (grp_na_count > 0))
```